lib/setTableAttrs.js

```diff
@@ -34,7 +35,14 @@ var cheerio = require('cheerio'),
             styleVal = $el.css(style);
 
             if (styleVal !== undefined) {
-                $el.attr(styleToAttrMap[style], styleVal);
+                $el.attr(
+                    styleToAttrMap[style],
+                    (attributesToRemovePxFrom.indexOf(style) > -1)
```

Comment:
This is a style issue but I think there is a better way to handle this. Maybe create a variable for it? I don't like including the ternary assignment here or splitting it into multiple lines like that.

Reply:
I'm not married to it. You prefer if statements to ternaries?

Reply:
Updated in master :)